### PR TITLE
Replace `scda` with `random.cdisc.data`

### LINKED ISSUE #154: [Feature Request]: remove `{scda*}` packages, and use `{random.cdisc.data}`
https://github.com/insightsengineering/teal.gallery/issues/3#issuecomment-1976672476

### PR DESCRIPTION
**Pull request by vedhav**
Closes #154

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
         run: |
           echo "BRANCH_NAME=dev" >> $GITHUB_ENV
           if [ "${{ matrix.channel }}" == "stable" ]; then
-            echo "BRANCH_NAME=main" >> $GITHUB_ENV
+            echo "BRANCH_NAME=154-use-rcd@main" >> $GITHUB_ENV
```
Comment (vedhav) [inline .github/workflows/deploy.yml:63]:
```suggestion
            echo "BRANCH_NAME=main" >> $GITHUB_ENV
```

Have to revert this before we merge. Also, remember to bring the changes to the `dev` branch.

**Comment by kartikeyakirar** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by kartikeyakirar — CHANGES_REQUESTED** [review]
@vedhav can you search and remove `scda` and `scda.2022`  from `tm_front_page` module used in all the apps.
e.g
https://github.com/insightsengineering/teal.gallery/blob/main/exploratory/app.R#L217-L225

**Comment by vedhav** [comment]
Thanks, done in 0900320a102a138e47fc3b6f199f8ab92034f42b

**Review by kartikeyakirar — APPROVED** [review]
@vedhav Great job, LGTM 🚀

**Pull request merged**